components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/utils/APIUtil.java

```diff
@@ -1404,6 +1405,18 @@ public static String getOpenAPIDefinitionFilePath(String apiName, String apiVers
                 apiName + RegistryConstants.PATH_SEPARATOR + apiVersion + RegistryConstants.PATH_SEPARATOR;
     }
 
+    public static String getIntrospectionQuery() throws APIManagementException {
```

Comment:
Add the method comment

Reply:
Resolved by 6f524e383ac97905ab494eff6a770c2ea09633a7